comps/vectorstores/langchain/pgvector/README.md

```diff
@@ -0,0 +1,21 @@
+# Start PGVector server
+
+## 1. Download Pgvector image
+
+```bash
+docker pull pgvector/pgvector:0.7.0-pg16
```

Comment:
This should be part of the BoM list, please fill in.

Reply:
Done